src/Components/Navbar.js

```diff
@@ -28,9 +30,6 @@ export default function NavBar() {
                         <Tab label="Manufacturers" value="3" />
                     </TabList>
                 </AppBar>
-                <TabPanel value="1"></TabPanel>
-                <TabPanel value="2"></TabPanel>
-                <TabPanel value="3"></TabPanel>
```

Comment:
Nice move removing these, since removing them doesn't change anything, they're better to be removed.